src/lib/operators/import.cpp

```diff
@@ -51,9 +53,37 @@ std::shared_ptr<const Table> Import::_on_execute() {
   auto table = std::shared_ptr<Table>{};
 
   switch (_file_type) {
-    case FileType::Csv:
-      table = CsvParser::parse(filename, _chunk_size, _csv_meta);
+    case FileType::Csv: {
+      if (static_cast<int>(static_cast<bool>(_csv_meta)) +
```

Comment:
I don't know how the others think about this, but I'd be fine just to prioritize the first definition source and not fail if there are multiple.

Reply:
It is!